TECHMANIA/Assets/Scripts/Paths.cs

```diff
@@ -66,7 +71,10 @@ public static string GetTrackRootFolder()
     {
         return trackRootFolder;
     }
-
+    public static string GetSaTrackRootFolder()
```

Comment:
It's not immediately clear that "sa" means "streaming assets", especially at call sites. Consider renaming to "GetStreamingTrackRootFolder", and the same for other methods below.